### ISSUE TITLE
call me maybe: timing off?

### ISSUE THREAD

**Issue by justinabrahms** [workshopper/goingnative#32]
The instructions to "Call me Maybe" say that `Sleep` takes milliseconds and `usleep` takes microseconds. We accept an integer from the command line which is total number of seconds.

Why is it that the reference solution is `Sleep(delay)` and not `Sleep(delay * 1000)` ?

**Comment by GochoMugo** [comment]
Related to this issue of timing, the reference solution and verification results seem misleading.

```bash
111✗ JavaScript code loads addon, invokes `delay(x, cb)` method and sleeps for x seconds
```
Wouldn't the reference solution then be:
```cpp
#ifdef _WIN32
Sleep(delayTime * 1000); // milliseconds
#else
usleep(delayTime * 1000 * 1000); // microseconds
#endif
```

**Comment by rvagg** [comment]
@GochoMugo this looks like a bug in the wording of exercise.js which should be changed to say _"x milliseconds", obviously wasn't updated in the last round of changes here

fixed in 279a0bf & published as 1.4.13, thanks!
